### PR TITLE
feat: add option to close overlay

### PR DESCRIPTION
**Pull request by anshumanv**
- [ ] This is a **bugfix**
- [x] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?
yet to 

### Motivation / Use-Case
add a button on the error overlay to manually allow closing the overlay

### Breaking Changes
no

### Additional Info
 fixes https://github.com/webpack/webpack-dev-server/issues/3361

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,8846 @@
+Arguments:
```
Comment (snitin315) [inline yarn-error.log]: This file should be in `.gitignore`

Comment (anshumanv) [reply yarn-error.log]: adding separate

**Review by alexander-akait — COMMENTED** [review]
Can we add `e2e/overlay.test.js` and add simple test?

**Comment by anshumanv** [comment]
yep lets add it

**Comment by alexander-akait** [comment]
@anshumanv I will write one couple tests today, so in future you will easy to add them more

**Comment by anshumanv** [comment]
yep big thanks @alexander-akait!

**Comment by alexander-akait** [comment]
I'm tired of these tests, it is more stable right now, but still very flaky, 80% times of my time I spend on tests instead writing code

**Pull request merged**